### PR TITLE
Launch RKE2 + EC2

### PR DESCRIPTION
**Pull request by btat**
Adds new tabs containing instructions sets (manual and cluster template) to launch RKE2 clusters on EC2.

### CODE REVIEW

File: content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/_index.md [hunk 1]
```diff
@@ -59,20 +62,81 @@ Add one or more node pools to your cluster. For more information about node pool
...
+### 2. Create your cluster
+
+1. Click **☰ > Cluster Management**.
+1. On the **Clusters** page, click **Create**.
+1. Toggle the switch to **RKE2/K3s**.
+1. Click **Amazon EC2**.
+1. Enter a **Cluster Name**.
+1. Create a machine pool for each Kubernetes role. Refer to the [best practices]({{<baseurl>}}/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools#node-roles-in-rke2) for recommendations on role assignments and counts.
+    1. For each machine pool, define the machine configuration. Refer to [the EC2 machine configuration reference]({{<baseurl>}}/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/ec2-machine-config/) for information on configuration options.
+1. Use the **Cluster Configuration** to choose the version of Kubernetes that will be installed, what network provider will be used and if you want to enable project network isolation.  For help configuring the cluster, refer to the [RKE2 cluster configuration reference.]({{<baseurl>}}/rancher/v2.6/en/cluster-admin/editing-clusters/rke2-config-reference/)
```
[the remark this answers is not in the document]
Comment (btat) [reply content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/_index.md]: Removed. Thanks!

**Review by markusewalker — APPROVED** [review]

**Review by sowmyav27 — APPROVED** [review]

File: content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/ec2-machine-config/_index.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+Enable access to [EC2 metadata](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html).
+
+### Use tokens for metadata
+
+Use [Instance Metadata Service Version 2 (IMDSv2)](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html), a token-based method to access metadata.
+
+### Add Tag
+
+Add metadata using [tags](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html) to categorize resources.
```
Comment (sowmyav27) [inline content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/ec2-machine-config/_index.md:78]: There are a few more fields after this
- Auto Replace
- Drain before Delete
- Kubernetes Node labels
- Taints

**Review by sowmyav27 — CHANGES_REQUESTED** [review]

File: content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/ec2-machine-config/_index.md (hunk 2 above)
Comment (snasovich) [reply content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/ec2-machine-config/_index.md:78]: @sowmyav27 , these would be general RKE2 features and not specific to infrastructure provider like EC2. So I think they should be specified in more generic RKE2 provisioning documentation.

File: content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/_index.md [hunk 3]
```diff
@@ -59,20 +62,81 @@ Add one or more node pools to your cluster. For more information about node pool
...
+1. Click **Amazon EC2**.
+1. Enter a **Cluster Name**.
+1. Create a machine pool for each Kubernetes role. Refer to the [best practices]({{<baseurl>}}/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools#node-roles-in-rke2) for recommendations on role assignments and counts.
+    1. For each machine pool, define the machine configuration. Refer to [the EC2 machine configuration reference]({{<baseurl>}}/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/ec2-machine-config/) for information on configuration options.
+1. Use the **Cluster Configuration** to choose the version of Kubernetes that will be installed, what network provider will be used and if you want to enable project network isolation. For help configuring the cluster, refer to the [RKE2 cluster configuration reference.]({{<baseurl>}}/rancher/v2.6/en/cluster-admin/editing-clusters/rke2-config-reference/)
+1. Use **Member Roles** to configure user authorization for the cluster. Click **Add Member** to add users that can access the cluster. Use the **Role** drop-down to set permissions for each user.
+1. Click **Create**.
+
+{{% /tab %}}
+{{% tab "RKE2 - Cluster Template" %}}
```
Comment (snasovich) [inline content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/_index.md:102]: I would recommend not splitting out "RKE2 - Cluster Template" case for each individual node provider as it's largely the same approach regardless of specific provider and is (at least at high level) is covered under https://rancher.com/docs/rancher/v2.6/en/admin-settings/cluster-templates/

File: content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/_index.md [hunk 4]
```diff
@@ -59,20 +62,81 @@ Add one or more node pools to your cluster. For more information about node pool
...
+
+### 3. Create your cluster using a cluster template
+
+1. Click **☰ > Cluster Management**.
+1. Under the **Use a Catalog Template to create a cluster** section, click **catalog-template**.
+1. Enter a name for the cluster.
+1. Select cloud credentials to use.
+1. Select the **Infrastructure Provider**. If you are using Rancher's [example cluster templates](https://github.com/rancher/cluster-template-examples), select `amazonec2`.
+1. Choose a **Kubernetes Version**.
+1. Configure your nodepools. For help with configurations, refer to the [EC2 Node Template Configuration](./ec2-node-template-config).
```
Comment (snasovich) [inline content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/_index.md:127]: Node Templates are not applicable to RKE2 provisioning case, so we don't want to reference this doc. This is likely a moot point anyway as we probably want to remove this whole section as per previous comment.

File: content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/ec2-machine-config/_index.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
+---
+title: EC2 Machine Configuration Reference
+weight: 2
+---
+
+For more details about EC2, nodes, refer to the official documentation for the [EC2 Management Console](https://aws.amazon.com/ec2).
```
Comment (snasovich) [inline content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/ec2-machine-config/_index.md:6]:
```suggestion
For more details about EC2 nodes, refer to the official documentation for the [EC2 Management Console](https://aws.amazon.com/ec2).
```

File: content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/ec2-machine-config/_index.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+### IAM Instance Profile Name
+
+The name of the [instance profile] used to pass an IAM role to an EC2 instance.
+
+## Advanced Options
+
+### AMI ID
+
+The [Amazon Machine Image](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) used to provision your cluster.
```
Comment (snasovich) [inline content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/ec2-machine-config/_index.md:36]:
```suggestion
The [Amazon Machine Image](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) used for the nodes in your cluster.
```

**Review by snasovich — CHANGES_REQUESTED** [review]

File: content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/_index.md (hunk 3 above)
Comment (snasovich) [reply content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/_index.md:102]: If we proceed with this, will need to update it for https://github.com/rancher/docs/pull/4066 (already merged), https://github.com/rancher/docs/pull/4085, and any other cloud providers docs we do.

Comment (btat) [reply content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/_index.md]: Section removed. Will do the same for the other infra sections.

File: content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/ec2-machine-config/_index.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+The [instance type](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-types.html), which determines the hardware characteristics, used to provision your cluster.
+
+### Root Disk Size
+
+Configure the size (in GB) for your [root device](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/RootDeviceStorage.html).
+
+### VPC/Subnet
+
+The [VPC](https://docs.aws.amazon.com/vpc/latest/userguide/configure-your-vpc.html) or specific [subnet](https://docs.aws.amazon.com/vpc/latest/userguide/configure-subnets.html), an IP range in your VPC, to your resources to.
+
```
Comment (anupama2501) [inline content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/ec2-machine-config/_index.md:27]: `to your resources to.` we can remove the to at the end or do we want to add anything after to?

Comment (btat) [reply content/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/ec2/ec2-machine-config/_index.md:27]: Typo fixed

**Review by snasovich — APPROVED** [review]

**Pull request merged**